rpc_jobs/rpc_upgrades.yml

```diff
@@ -0,0 +1,97 @@
+### tests rpc-upgrades to <scenario> leaped from <action>
+- project:
+    name: "rpc-upgrades-aio-newton-head"
+    repo_name: "rpc-upgrades"
+    repo_url: "http://github.com/rcbops/rpc-upgrades"
+    series: "master"
+    branches:
+     - "master"
+    image:
+      - aio:
+          FLAVOR: "performance2-15"
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "swift"
+    action:
+      - "liberty_to_newton_leap"
+      - "r12.2.8_to_newton_leap"
+#      - "r12.2.5_to_newton_leap"
+#      - "r12.2.2_to_newton_leap"
+#      - "r12.1.2_to_newton_leap"
+#      - "kilo_to_newton_leap" 
+    jira_project_key: "RLM"
+    jobs:
+      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
+      - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+- project:
```

Comment:
This could be squashed into the above, right?

Reply:
It could, but I might want to leverage the name to differentiate the seperate targets, newton release vs newton head vs aio vs mnaio.  It shouldn't hurt anything being split out should it?

Reply:
There's no problem w/ separate projects if that's what works best.